### DISCUSSION TITLE
What happened to release 3.6.3 ?

### DISCUSSION THREAD

**Discussion by stbuerger** [emsesp/EMS-ESP32#1466] [General]
Looking today (20231126) in/at "releases" I can only see stable releases 3.6.2 - and then next one is - 3.6.4.
I would have expected a 3.6,3  in between. Or do I miss something here ?

**Comment by proddy** [comment]
well spotted!

3.6.3 had a serious bug in it, so we took it offline and patched, which is now 3.6.4. We use semantic versioning so MAJOR.MINOR.PATCH
